lib/lambda_ethereum_consensus/event_handlers/block_handler.ex

```diff
@@ -0,0 +1,15 @@
+defmodule LambdaEthereumConsensus.Handlers.BlockHandler do
```

Comment:
Alternatively, we could also have one single event handler with one function per topic like:
`def handle_message("beacon_block", %SignedBeaconBlock{message: block}) do`